CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # Changelog
 
+### Web3
+
+- Added new design for NFT spam management in Brave Wallet. ([#8205](https://github.com/brave/brave-ios/issues/8205))
+- Added bottom tab bar for Brave Wallet. ([#8166](https://github.com/brave/brave-ios/issues/8166))
+- Added the ability to group NFTs in Brave Wallet. ([#8073](https://github.com/brave/brave-ios/issues/8073))
+- Added new design for the Portfolio page. ([#8072](https://github.com/brave/brave-ios/issues/8072))
+- Added method to return "Swap" protocol fees. ([#7924](https://github.com/brave/brave-ios/issues/7924))
+- Added support for Sign-In with Ethereum (EIP-4361). ([#7827](https://github.com/brave/brave-ios/issues/7827))
+- Added support for EIP-6963. ([#7779](https://github.com/brave/brave-ios/issues/7779))
+- Added the ability for users to navigate to ENS domains. ([#6225](https://github.com/brave/brave-ios/issues/6225))
+- Added the ability to select a specific RPC URL as active for Brave Wallet. ([#5902](https://github.com/brave/brave-ios/issues/5902))
+- [Security] Updated Brave Wallet password policy. ([#6093](https://github.com/brave/brave-ios/issues/6093))
+- Implemented "Bridge to Aurora" for Brave Wallet. ([#5785](https://github.com/brave/brave-ios/issues/5785))
+- Updated Portfolio settings page in Brave Wallet. ([#8089](https://github.com/brave/brave-ios/issues/8089))
+- Updated button to create wallet to be disabled when wallet is already being created. ([#8357](https://github.com/brave/brave-ios/issues/8357))
```

Comment:
change to `Disabled "Continue" button on create wallet screen when wallet is already being created`